src/AWS.Deploy.Common/DeploymentManifest/DeploymentManifestModel.cs

```diff
@@ -11,11 +11,13 @@ namespace AWS.Deploy.Common.DeploymentManifest
     /// </summary>
     public class DeploymentManifestModel
     {
-        [JsonProperty("deployment-projects")]
-        public List<DeploymentManifestEntry> DeploymentManifestEntries { get; set; }
+        public List<LastDeployedStack>? LastDeployedStacks { get; set; }
 
-        public DeploymentManifestModel(List<DeploymentManifestEntry> deploymentManifestEntries)
+        public List<DeploymentManifestEntry>? DeploymentManifestEntries { get; set; }
```

Comment:
We already have JsonProperty called "deployment-projects". But its being removed as part of this PR

Reply:
@96malhar Yes, I asked in previous comment to make the name follow the same casing pattern. I'm good with using deployment project but we should probably use `DeploymentProjects` instead of `deployment-projects`.

Reply:
Fixed